### PR TITLE
Explicitly depend on database migrations and IAM during deployment

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
Explicitly depend on database migrations and IAM during deployment
```

/assign @chaodaiG @icco

### CODE REVIEW

File: terraform/service_apiserver.tf [hunk 1]
```diff
@@ -149,8 +149,19 @@ resource "google_cloud_run_service" "apiserver" {
 
   depends_on = [
     google_project_service.services["run.googleapis.com"],
+
     google_secret_manager_secret_iam_member.apiserver-db,
+    google_kms_key_ring_iam_member.kms-signerverifier,
```
Comment (chaodaiG) [inline terraform/service_apiserver.tf:154]: Is it possible to define this collection somewhere and be referenced here?

Comment (sethvargo) [reply terraform/service_apiserver.tf:154]: That would certainly be nice, but no.

Comment (jeremyfaller) [reply terraform/service_apiserver.tf:154]: I am consistently disappointed in aspects of this tool. Either we're doing something wrong, or there's an opportunity here.

**Pull request merged**